package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-cli",
-  "version": "4.2.0",
+  "version": "5.0.0",
```

Comment:
Is this to prevent tests breaking downstream?

Reply:
sorry for replying late 🙈 

> prevent tests breaking downstream

Exactly.